src/프린터.js

```diff
@@ -0,0 +1,24 @@
+function solution(priorities, location) {
+  let maxNumber = 0;
+  let maxNumberIndex = 0;
+  let myturn = 1;
+  let newPriorities;
+
+  for (let i = 0; i < priorities.length; i++) {
+    maxNumber = Math.max(priorities[i], maxNumber);
+  }
```

Comment:
```suggestion
  let maxNumber = Math.max(...priorities);
```
이렇게도 최대값을 구하는게 가능할 꺼 같습니다!